config/config.go

```diff
@@ -474,6 +482,30 @@ func (c *ServersetSDConfig) UnmarshalYAML(unmarshal func(interface{}) error) err
 	return checkOverflow(c.XXX, "serverset_sd_config")
 }
 
+// MarathonSDConfig is the configuration for services running on Marathon.
+type MarathonSDConfig struct {
+	ServerScheme   string   `yaml:"server_scheme,omitempty"`
```

Comment:
I was more thinking that the `Servers` string may contain a scheme or not. This is easier to configure and handles the theoretical possibility that different servers have different schemes.

So "http://foo.bar" and "foo.bar" would both end up as "http://foo.bar".